src/components/badges/smallElectric.tsx

```diff
@@ -0,0 +1,34 @@
+import React, { FC } from "react"
+
+import { BadgeProps } from "./types"
+import BaseBadge from "./base"
+import ElectricIcon from "../../assets/dist/electric"
+
+export const SmallElectricBadge: FC<BadgeProps> = ({
+  language,
+  size = "large",
+  tooltipContent,
+}) => {
+  const title = {
+    de: "Elektro",
+    fr: "Électrique",
+    it: "Elettrica",
+    en: "Electric",
+  }
+
+  return (
+    <div className="text-white">
+      <BaseBadge
+        icon={<ElectricIcon width="24" height="24" />}
+        tooltipContent={tooltipContent}
+        size="small"
+        withText={size === "large"}
```

Comment:
I like `withText` better as well but in order to remain consistent we would need to replace `size` by `withText` for each badge, is it fine for you ? 
It involves a breaking change, it will take longer for the PR to be merged. I am totally fine doing this, I just want to make sure we are on the same page 🙂

Reply:
glad you're seeing the same challenges as I had in mind too. 

it can also be done in separate steps (release an inconsistent api, make it consistent again in a separate Tech20). I feel that it came up a couple of times now in separate PRs, so I'd lean towards making the effort. I'll leave the decision up to you